tuist/tuist
# Add support for embedded SPM dependencies

Resolves https://github.com/tuist/tuist/issues/6683

### Short description 📝

This pull request adds support for embedded SPM dependencies through Xcodebuild. 

> **NOTE:** This pull request depends on change in `XcodeGraph` https://github.com/tuist/XcodeGraph/pull/57

### How to test the changes locally 🧐

* Add SwiftPackage with `.dynamic` product type as dependency by using `.package`
* Add `.packageProduct(name: "Name", type: .runtimeEmbedded)` to dependencies of the target
* Generate project

**Expected result:** this dependency marked as `embedded`

### Contributor checklist ✅


### Reviewer checklist ✅
